### PR TITLE
Adds Hot Toddy reagant and recipe 

### PR DESCRIPTION
**Pull request by Carbadox**
## About the PR 
Adds the hot toddy cocktail and recipe into the game.

## Why's this needed? 
To expand the options for cocktails in the game

## Changelog
```
(u)Carbadox
(+)Added the Hot Toddy cocktail into the game as well as a new recipe
```

### CODE REVIEW

**Comment by flrsh** [comment]
For the sake of convenience, could you move your code chunks to next to where the bulk of the drink recipes + reagents are? 

Also, in the future, you can tag your PR so that the bot will autolabel it!

**Comment by ZeWaka** [comment]
If you want to fix travis failing, you'll have to pull from master.

**Comment by ZeWaka** [comment]
The description says it might help cure sickness, why doesn't it do so?

**Comment by Carbadox** [comment]
Because it's not supposed to. The description is only flavour text and even then it says it's to relieve the symptoms of, not cure. I don't think there's any IRL proof that it metabolically helps fight off sickness. It's more likely that it does the exact opposite as alcohol is known to weaken the immune system. If need be I can modify the text to be more consice but I believed what I had written was already sufficient and specific.

**Comment by Gannets** [comment]
I think you've gotta pick a side between realism and expected videogame behaviour and have the description and the effect line up.

Chicken soup doesn't do much for me irl but it can cure a spaceman of their minor illnesses.

**Comment by Carbadox** [comment]
I'll rephrase what I said then. My intention was only to add a new cocktail to the game with no other gimmicks or attributes to it besides it just being a cocktail. I'll change the description to be less misleading once I get around to it.

File: code/modules/chemistry/Chemistry-Recipes.dm [hunk 1]
```diff
@@ -4013,4 +4013,14 @@ datum
 			required_reagents = list("poor_cement" = 1, "silicon_dioxide" = 5, "water" = 1)
 			result_amount = 7
 
+		hot_toddy
+			name = "Hot Toddy"
+			id = "hottoddy"
+			result = "hottoddy"
+			required_reagents = list("bourbon" = 1, "tea" = 1, "juice_lemon" = 1)
+			result_amount = 3
+			mix_phrase = "The drink suddenly fills the room with the a fetive aroma."
```
Comment (mordent-goonstation) [inline code/modules/chemistry/Chemistry-Recipes.dm:4022]: Typo "fetive" -> "festive"?

File: code/modules/chemistry/Chemistry-Recipes.dm [hunk 2]
```diff
@@ -4013,4 +4013,14 @@ datum
 			required_reagents = list("poor_cement" = 1, "silicon_dioxide" = 5, "water" = 1)
 			result_amount = 7
 
+		hot_toddy
+			name = "Hot Toddy"
+			id = "hottoddy"
+			result = "hottoddy"
+			required_reagents = list("bourbon" = 1, "tea" = 1, "juice_lemon" = 1)
```
Comment (mordent-goonstation) [inline code/modules/chemistry/Chemistry-Recipes.dm:4020]: Consider adding `sugar` to this (or `honey`, but I don't think we have that in reagent form in any easily obtainable way), to be a bit truer to the actual recipe?

**Comment by mordent-goonstation** [comment]
Not tying this comment to a specific line of code, but I think to be a *hot* toddy it should have to be, y'know... hot.

You could have a temperature requirement, and (like with, say, water/ice) if it drops too cool it could become a cold toddy.

File: code/modules/chemistry/Chemistry-Recipes.dm (hunk 2 above)
Comment (Tarmunora) [reply code/modules/chemistry/Chemistry-Recipes.dm:4020]: Honey is obtainable from bees. Botany will usually have a surplus

**Comment by Carbadox** [comment]
gonna open a new PR once I work this out

**Pull request closed without merging**